web/.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
+build/
```

Comment:
I decided to separate `building` (TypeScript -> JavaScript) and `packing` (one file bundle) to use two different folders.